src/blob/blob_tests.rs

```diff
@@ -174,7 +174,7 @@ async fn test_selfavatar_outside_blobdir() {
     let avatar_blob = t.get_config(Config::Selfavatar).await.unwrap().unwrap();
     let avatar_path = Path::new(&avatar_blob);
     assert!(
-        avatar_blob.ends_with("1e08d1c9398297c21dd3820f7db2324.jpg"),
```

Comment:
Do we really need to check for the exact hash here? I we want, e.g. to know if the JPEG encoder suddenly changes and gives (slightly) different images, it's sufficient to have the only test for that, modifying two tests every time is annoying

Reply:
idk, i did not wrote the test, just adapted - and i did not find it much work

it is not that often that the hashes need to be adapted. up to now, it was only when we actually change sth in our code, there were no test adaptions needed because of JPEG encoder so far. i am fine with it